main/repl/timerServices.md

```diff
@@ -21,54 +21,48 @@ Both timer services let you:
-services is whether you preface their identical command set with
-`localTimerService` or `chainTimerService`. However, note that
-the local and chain versions of a method may use different time bases.
+Both timer services present an identical API, but apply a different interpretation to
+the following data types.
 
-Before looking at the timer service methods, we need to cover four related objects.
+### `Timestamp`
 
-### `Timestamp` object
+A `BigInt` representing an absolute instant in time in a `TimerService`, using units and a basis dependent upon the service.
 
-An `Integer`, an absolute individual stamp returned by a `TimerService`.  Note that different
-timer services may have different interpretations of actual Timestamp values.
+### `RelativeTime`
 
-### `RelativeTime` object
-
-An `Integer`, the difference between two `Timestamps`.  Note that
-different timer services may have different interpretations of actual `RelativeTime` values.
+A `BigInt` representing the difference between two `Timestamp`s of the same `TimerService`.
 
 ## TimerService methods
 
 The following methods provide basic TimerService functionality.
 
 ### `E(home.<chain or local>TimerService).getCurrentTimestamp()`
-- Returns: A monotonically increasing timestamp for the chain or local ag-solo.
-
-The current block might be executed more than once in case of restart or replay.
-But each time it will start from the same state and receive the same inputs. 
-Since this is repeatable computation, the same computation can be run in various
-locations to cross-verify. So each time, the `currentTimeStamp` will be the same, 
-even if we're running the computation a minute or a month later. It's expressed 
-as a Unix epoch time in milliseconds or similar.
-```
+- Returns: `{ Promise<Timestamp> }`
+
+Returns a promise that resolves with the current timestamp.
+
+```js
 command[1] E(home.chainTimerService).getCurrentTimestamp()
 history[1] 1632170301n // in seconds since epoch
 command[2] E(home.localTimerService).getCurrentTimestamp()
 history[2] 1632170301546n // in milliseconds since epoch
 ```
 
+A chain block might be executed more than once in case of restart or replay or
+cross-verification, but each such execution will receive the same input state
+and observe the same response, even if running a minute or a month later.
+
 ### `E(home.<chain or local>TimerService).delay(delay)`
 - `delay` `{ RelativeTime }`
-- Returns: `{ Promise<Timestamp>}`
+- Returns: `{ Promise<Timestamp> }`
 
 Returns a promise that resolves with the current timestamp after the `delay`
-relative time has passed.
+has elapsed.
 
-```
-command[32] E(home.localTimerService).delay(3_000n)
-history[32] unresolved Promise
-history[32] 1632170321864n // about 3 seconds later, in milliseconds since epoch
+```js
+command[3] E(home.localTimerService).delay(3_000n)
+history[3] unresolved Promise
+history[3] 1632170321864n // about 3 seconds later, in milliseconds since epoch
```

Comment:
any particular reason to change these numbers?

Reply:
No big reason, just for continuity with the previous block and consistency with sibling documents.